chartfx-dataset/src/test/java/de/gsi/dataset/spi/EditableDataSetTests.java

```diff
@@ -0,0 +1,337 @@
+package de.gsi.dataset.spi;
+
+import static de.gsi.dataset.DataSet.DIM_X;
+import static de.gsi.dataset.DataSet.DIM_Y;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.gsi.dataset.DataSetError;
+import de.gsi.dataset.EditConstraints;
+import de.gsi.dataset.EditableDataSet;
+
+/**
+ * Tests the specific data set implementations for adherence to the EditableDataSet interface.
+ *
+ * @author rstein
+ */
+public class EditableDataSetTests {
+    private static final Logger LOGGER = LoggerFactory.getLogger(EditableDataSetTests.class);
+    protected static final int N_POINTS_INSERT_FRONT = 3;
+    protected static final int N_POINTS_ADD_BACK = 5;
+    protected static final double[] testCoordinates = { 1.0, 2.0 };
+    protected static final double[] errorCoordinates = { 0.1, 0.2, 0.3, 0.4, 0.5, 0.6 };
+
+    @Test
+    public void testCommonDataSetImplementations() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=4728036)